share/commands/agent-info

```diff
@@ -17,6 +17,12 @@ HASH=$(echo "${HASH}" | sed -e 's/.*= //')
 POLICY=$(cat /var/rudder/cfengine-community/policy_server.dat 2>/dev/null)
 [ $? -ne 0 ] && POLICY="Not yet configured"
 
+POLICY_FETCHED=$( stat -c %y /var/rudder/cfengine-community/last_successful_inputs_update 2> /dev/null )
+[ $? -ne 0 ] && POLICY_FETCHED="Not updated yet" || POLICY_FETCHED=$( cut -d. -f1 <<<"$POLICY_FETCHED"  )
```

Comment:
We prefer the form "${POLICY_FETCHED}" for variables.
And '<<<' is a bashism, you should use 'echo |' instead